tests/unit/seed/wheels/test_periodic_update.py

```diff
@@ -559,51 +602,58 @@ def rel_date():
 
     assert download_wheel.call_count == 2
     assert release_date.call_count == 2
-
     assert read_dict.call_count == 1
     assert write.call_count == 1
 
 
-def test_download_manual_stop_at_first_usable(tmp_path, mocker, freezer):
+def test_download_periodic_stop_at_first_usable2(tmp_path, mocker, freezer):
```

Comment:
Why the 2 suffix?

Reply:
there's already a `test_download_periodic_stop_at_first_usable` test and I didn't see how to do this as a parametrize test easily so adding a suffix `2` on the second one.

Reply:
Instead of magic numbers, I generally add to the test name what differentiates them :thinking: for future reference.